src/app/view/FlagshipUI/FlagshipUIService.ts

```diff
@@ -0,0 +1,174 @@
+import { FlagshipUI } from 'cozy-intent'
+import Minilog from 'cozy-minilog'
+
+import { EventEmitter } from 'events'
+
+import { applyFlagshipUI } from '/libs/intents/setFlagshipUI'
+
+const log = Minilog('🖌️ FlagshipUIService')
+
+/**
+ * Declare all screens ZIndex here
+ *
+ * This array should be used everytime we call useFlagshipUI to apply a ZIndex to the component
+ * All ZIndexes are grouped here to ensure no screen overlap
+ */
+export const ScreenIndexes = {
```

Comment:
For me all this is declarative config and shouldn't really be in the same place

Reply:
yes a new file as I don't see an existing one filling that role. but maybe it can be handled later as it's non critical

Reply:
Done here : https://github.com/cozy/cozy-flagship-app/pull/1082/commits/7b2a6c9394ef67df5b5b4b74b361742eb5bd9872